### DISCUSSION TITLE
Help | I can't build own twrp

### DISCUSSION THREAD

**Discussion by FTS427** [SebaUbuntu/TWRP-device-tree-generator#187] [Q&A] [answered]
Please look at the screenshot below, how can I solve this problem?

![build_log.png](https://github.com/twrpdtgen/twrpdtgen/assets/122330825/0e1ecd26-29fa-4171-be0e-bbd2af571240)

I wanted to use Github Action to build twrp, but it seems that there is something wrong with the building commands.

A useless argument "-dtb" appeared instead of a correct "-dt", it may be what caused the whole building process didn't work.

What need I change to build twrp correctly?

(I'm a new and also a Chinese, so the description above may be strange, sorry)

**Comment by SebaUbuntu** [comment, marked as answer]
You're using the wrong TWRP version, use a newer branch

**Reply by FTS427** [reply]
Thanks
